test/TestLineNumber.py

```diff
@@ -0,0 +1,49 @@
+# Copyright (c) 2013-2014 Albin Vass <albin.vass@gmail.com>
```

Comment:
Did you really create this test class 7 years ago?

Reply:
Oops ;)